### PR TITLE
adjust the logic of adding course manually

### PR DESCRIPTION
**Pull request by AInfinity-LilacDream**
adjust the logic of adding course manually so that courses on Sunday will be displayed correctly

### CODE REVIEW

**Comment by w568w** [comment]
A few thoughts:

- Your commit message format does not conform to the required guidelines[^1][^2]. Please reword it to something like `fix: ...` or `change: ...`[^3];
- Before modifying (especially reverting) someone else's changes, **ask the person who made the change first**: (e.g., in this issue) “*Why did you treat the weekend differently? Was there some consideration for fixing other bugs? Is there a risk that my roll-back will cause other cascading problems?*” Avoid letting a narrow focus on fixing a specific issue prevent you from seeing other, larger problems, which could make things worse![^4]


[^1]: https://github.com/DanXi-Dev/DanXi/commit/3262b64b61b8670c8879663df73ea1a140804ff5#comments
[^2]: https://ruby-china.org/topics/15737
[^3]: I have also given some instructions about the best practice in Feishu work group "旦夕 App". Please read the pinned messages there first.
[^4]: With Intellij IDEs, you can list who has ever changed the lines with [View Selection History](https://www.jetbrains.com/help/idea/investigate-changes.html#view_selection_history).

**Review by w568w — CHANGES_REQUESTED** [review]
As above.

**Comment by AInfinity-LilacDream** [comment]
Too busy last week，just working on it，at most four days required.



---Original---
From: ***@***.***&gt;
Date: Tue, Oct 22, 2024 11:43 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [DanXi-Dev/DanXi] adjust the logic of adding course manually (PR#427)




 
@w568w requested changes on this pull request.
 
As above.
